serving/pom.xml

```diff
@@ -248,11 +253,9 @@
       <scope>test</scope>
     </dependency>
 
-    <!-- TODO: fix version discrepancy with managed version -->
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>2.23.0</version>
```

Comment:
A recent Byte Buddy is needed for some functionality of Mockito with Java 11, which is probably why we already have a 1.9.x version set in root POM dependency management. For some reason this older version of Mockito overriding in a couple of modules breaks even though the new Byte Buddy version should be forced, specifically when JaCoCo instrumentation is running. Letting these use newer Mockito set in dependency management already fixed it. [This seems to be a thing][1].

It appears there is no issue with the newer Mockito on these snowflake modules anymore anyway, so good to drop the special cases.

[1]: https://github.com/mockito/mockito/issues/1483#issuecomment-615477170